src/dashboard/tests/test_transfer.py

```diff
@@ -0,0 +1,56 @@
+import os
```

Comment:
Something I am considering more in my testing, although I haven't had much opportunity to practice it lately, is making sure there are some Unicode strings incorporated into tests so that there is good coverage for folks wanting to use AM in different languages. You'd need an encoding declaration `# -*- coding: utf-8 -*-` first. Up to you if you'd like to add some strings in the below form to test against.